content/docs/details/github-enterprise/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Configuring GitHub Enterprise
+publishedDate: '2022-07-28T21:00:00.0Z'
+description: How to connect the Roadie catalog to GitHub Enterprise.
+---
+
+## Introduction
+
+This tutorial will guide you through the steps required to connect Roadie to your GitHub org in GitHub Enterprise.
+
+## Prerequisites
+
+- Please let the Roadie support team know about your intention to configure GitHub enterprise, because we need to do some manual steps to enable it for you.
+- You will need to ensure that the GitHub Enterprise instance is available to our IP addresses published [here](/docs/details/allowlisting-roadie-traffic/): 
+
+## Step 1: Create an OAuth App in GitHub
+
+Create an OAuth App in GitHub https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app and generate a secret key. Keep note of the client ID and secret key.
```

Comment:
Do we need to tell them what scopes to request?

Reply:
Same for the PAT?

Reply:
The Oauth app does not have scopes. The PAT token does though.

Reply:
ah, right! it was the normal GH app 👍  thanks

Reply:
done.